### PR TITLE
git-annex 5.20150113

### PR DESCRIPTION
**Pull request by carlmod**

### CODE REVIEW

File: Library/Formula/git-annex.rb [hunk 1]
```diff
@@ -23,6 +23,7 @@ class GitAnnex < Formula
   depends_on "libidn"
   depends_on "gnutls"
   depends_on "gmp"
+  depends_on "wget"
```
Comment (mikemcquaid) [inline Library/Formula/git-annex.rb]: Is this a runtime or build-time dependency? If run-time: what for?

Comment (carlmod) [inline Library/Formula/git-annex.rb]: It is a runtime dependency used to transfer files from different so called special remotes. From reading the documentation it seems that either curl or wget should be needed, but that wget recommended. So ideally the dependency should be changed to :recommended. 

However, the latest version does not seem to build without an installed wget. I just reported this as a [bug](http://git-annex.branchable.com/bugs/Build_fails_when_no_wget_avalible/) upstreams.

Comment (mikemcquaid) [inline Library/Formula/git-annex.rb]: Let's make it `:build` for now as `curl` ships with OSX.

Comment (mikemcquaid) [inline Library/Formula/git-annex.rb]: And can you link the bug as a Ruby comment in the formula? Thanks!

**Comment by carlmod** [comment]
Noted that wget is just a build dependency for this particular version, and linked the now fixed upstreams bug.

**Pull request closed without merging**